tests/test_alara.py

```diff
@@ -647,3 +649,79 @@ def test_phtn_src_energy_bounds():
                          1.00E7, 1.20E7, 1.40E7, 2.00E7]
 
     assert_array_equal(e_bounds, expected_e_bounds)
+
+
+def test_alara_responses_output_zone():
+    # test decay_heat
+    response = ['decay_heat', 'specific_activity']
+    exp_response_code = \
+"""output zone
+      total_heat
+      specific_activity
+end"""
```

Comment:
Can the all-the-way-to-the-right indentation be avoided? It will probably fail an autopep8 check.

Reply:
Do you mean all-the-way-to-the-left?  I'm not sure there is a better way to generate output with appropriate indentation?